node/src/cli.rs

```diff
@@ -1,19 +1,20 @@
 use sc_cli::RunCmd;
-use structopt::StructOpt;
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, clap::Parser)]
 pub struct Cli {
-	#[structopt(subcommand)]
+	#[clap(subcommand)]
```

Comment:
Clap?

Reply:
Clap is a Rust crate for CLI, upstream has started using it.